assessments/projects/cloudevents/self-assessment.md

```diff
@@ -0,0 +1,1063 @@
+# CloudEvents self-assessment
+
+September 26, 2023
+
+Authors: Igor Rodrigues (@Igor8mr), Matthew Gong (@MatthewZGong), Kushal Kothari
+(@Kushal-kothari) and Devyani Bairagya (@devyani-14).
+
+Contributors/Reviewers: Pranava Kumar Vemula (@Rana-KV), Doug Davis (@duglin),
+Eddie Knight (@eddie-knight) and Ragashree (@ragashreeshekar).
+
+This document elaborates and explores the design goals for
+[CloudEvents](https://cloudevents.io/) as well as a security assessment of the
+software. It was written in collaboration with the CloudEvents team. All
+communications with CloudEvents were recorded on a [separate
+document](CE-maintainers-communications.md).
```

Comment:
I like that you put this here! But since we removed the comms file, we should restructure the paragraph.

```suggestion
This document was written in collaboration with the CloudEvents team to elaborate on and explores the design goals for
[CloudEvents](https://cloudevents.io/) through a limited security assessment of the software.
```

Reply:
Thank you, I committed the suggestion.